btb_phylo.py

```diff
@@ -238,15 +239,17 @@ def append_multi_fasta(s3_bucket, s3_key, outfile):
     """
     # temp directory for storing individual consensus files - deleted when function
     # returns
```

Comment:
The above comment just needs to be adjusted or removed because no longer using a tempfile

Reply:
Good spot